packages/common/src/scopeSupportFacets/scopeSupportFacetInfos.ts

```diff
@@ -124,6 +124,21 @@ export const scopeSupportFacetInfos: Record<
     description: "A named method declaration in a class",
     scopeType: "namedFunction",
   },
+  "namedFunction.method.iteration.class": {
+    description: "Iteration scope for named functions: class bodies",
+    scopeType: "namedFunction",
+    isIteration: true,
+  },
+  "namedFunction.iteration": {
```

Comment:
@pokey There's no indicator in the name or description what this iteration scope should be. Shouldn't this be `namedFunction.iteration.block`?

Reply:
Yeah so the issue is that we don't necessarily want the iteration scope for functions to always include blocks. For example, that's not what we do in typescript. I added this as a catch-all so that I could include blocks for css. Might be worth discussing?

Reply:
We can us gore had and merge this because we can always update later?

Reply:
Slightly mixed feelings about merging in facets that we will want to change, as I think it could be a bit painful to rename them, especially as they get used more

Reply:
Possibly. Should be punt on this then?

Reply:
Hmm. Would be good to get this PR in, but I think it's going to be easier to get this one right, as we use this facet for this PR. I'd be inclined to discuss Sunday

Reply:
Lets

Reply:
On second thought, let's get this thing in. I filed https://github.com/cursorless-dev/cursorless/issues/2375; let's discuss Sunday